src/ui.ts

```diff
@@ -208,47 +207,43 @@ public nodes: Nodes;
 
   /**
    * Shows uploading preloader
-   *
    * @param {string} src - preview source
    * @returns {void}
    */
-  showPreloader(src: string): void {
+  public showPreloader(src: string): void {
     this.nodes.imagePreloader.style.backgroundImage = `url(${src})`;
 
     this.toggleStatus(UiState.Uploading);
   }
 
   /**
    * Hide uploading preloader
-   *
    * @returns {void}
    */
-  hidePreloader(): void {
+  public hidePreloader(): void {
```

Comment:
are you sure all method should be public?

Reply:
still actual

Reply:
where the issue still remains?